From a122f67365dc593c362d00dfc8a29f737c6e3c51 Mon Sep 17 00:00:00 2001
Message-Id: <a122f67365dc593c362d00dfc8a29f737c6e3c51.1625836942.git.agx@sigxcpu.org>
From: =?UTF-8?q?Guido=20G=C3=BCnther?= <agx@sigxcpu.org>
Date: Fri, 9 Jul 2021 14:55:43 +0200
Subject: [PATCH] play: Emit correct signal

SIGNAL_MEDIA_INFO_UPDATED should be emitted on media info changes,
not SIGNAL_VIDEO_DIMENSIONS_CHANGED.
---
 gst-libs/gst/play/gstplay-signal-adapter.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gst-libs/gst/play/gstplay-signal-adapter.c b/gst-libs/gst/play/gstplay-signal-adapter.c
index cc5f0e0ce4..cf8b57e198 100644
--- a/gst-libs/gst/play/gstplay-signal-adapter.c
+++ b/gst-libs/gst/play/gstplay-signal-adapter.c
@@ -168,7 +168,7 @@ gst_play_signal_adapter_emit (GstPlaySignalAdapter * self,
       GstPlayMediaInfo *media_info;
       gst_structure_get (message_data, GST_PLAY_MESSAGE_DATA_MEDIA_INFO,
           GST_TYPE_PLAY_MEDIA_INFO, &media_info, NULL);
-      g_signal_emit (self, signals[SIGNAL_VIDEO_DIMENSIONS_CHANGED], 0,
+      g_signal_emit (self, signals[SIGNAL_MEDIA_INFO_UPDATED], 0,
           media_info);
       g_object_unref (media_info);
       break;
-- 
2.30.2

